src/modules/account/components/account-deposit/account-deposit.jsx

```diff
@@ -48,6 +64,16 @@ export default class AccountDeposit extends Component {
           </button>
         </div>
       )
+      airSwapConverter = (
+        <div className={Styles.AccountDeposit__shapeShiftButton}>
+          <button
+            onClick={e => airSwapOnClick(e)}
+            value={'https://shapeshift.io/shifty.html?destination=' + address + '&output=REP'}
```

Comment:
Hm, is this intended to be a ShapeShift link...?

Reply:
fixed, thank you.